zerver/webhooks/slack/view.py

```diff
@@ -204,12 +226,14 @@ def api_slack_webhook(
     if event_type != "message":
         raise UnsupportedWebhookEventTypeError(event_type)
 
+    check_token_access(slack_app_token)
```

Comment:
Is this call useful? I think it's probably undesirable to do the part of it that makes an API request to check if the token is working every time this webhook is hit; that can add significant latency to the integration. I'm also not sure the set of required scopes would be the same as those needed for the data export tokens, so it probably makes sense to make that an explicit parameter?

Probably we should just call this on the challenge code path, and just have the other code paths that make Slack API requests handle "this token was deactivated" class of exceptions properly.

This can probably be a follow-up, but should be a near-term one.